app/models/article.js

```diff
@@ -0,0 +1,16 @@
+import DS from "ember-data";
+
+const { attr, belongsTo } = DS;
+
+export default DS.Model.extend({
+    title: attr(),
```

Comment:
JSDoc/[ESDoc](https://esdoc.org/manual/usage/tags.html).  While we're not using it to actually generate documentation, I find it to be a nice way to write comment docs even without it. My syntax highlighting makes it pretty and it's easy to figure out what's going on.